### PR TITLE
updated missing code reference to upload_with_media

### PR DESCRIPTION
**Pull request by aschmid**
because the update_with_media action was deprecated i updated the statuses case to reflect those changes

### CODE REVIEW

File: twitter.js [hunk 1]
```diff
@@ -216,9 +216,12 @@ Twitter.prototype.statuses = function(type, params, accessToken, accessTokenSecr
 			delete params.id;
 			method = "POST";
 			break;
-		case "update_with_media":
-			this.updateWithMedia(params, accessToken, accessTokenSecret, callback);
+		case "upload_media":
+			this.uploadMedia(params, accessToken, accessTokenSecret, callback);
 			return;
+		case "update_with_media":
+			callback("Upload with media type is deprecated. Use 'upload_media' instead");
```
Comment (UnifiedInc) [inline twitter.js]: The word "deprecated" would be inaccurate given the code will not work. Suggest changing to "has been removed."

Comment (aschmid) [inline twitter.js]: updated and pushed

**Comment by UnifiedInc** [comment]
My suggestion may have been poorly thought out. Looking at the code it seems the only thing that's changed is the function's name. Why not throw the deprecation warning and call the new function for backwards compatibility?

**Comment by reneraab** [comment]
LGTM. Sorry, I totally forgot about this pull request :see_no_evil:

**Comment by aschmid** [comment]
thats not really what happened. the function update_with_media was creating a tweet with a media object in one call while now you have to first upload the media to twitter and then use the id to create the post.

**Comment by reneraab** [comment]
I think it doesn't really matter whether you call it removed or deprecated... It was "removed" in the code (i.e. there is no functionality anymore) and shouldn't (can't) be used anymore (with node-twitter-api).

**Comment by aschmid** [comment]
alright sounds good to me

**Pull request merged**